docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -158,9 +158,24 @@ spec:
 ```
 ## Uprobes
 
-{{% pageinfo %}}
-This hook point method lacks documentation, see [issue #878](https://github.com/cilium/tetragon/issues/878).
-{{% /pageinfo %}}
+## Uprobes
+
+Uprobes are similar to kprobes, but they allow you to dynamically hook into any user-space function and execute BPF code. Uprobes are also tied to the binary version of the user-space program, so they may not be portable across different versions or architectures.
+
+To use uprobes, you need to specify the path to the executable or library file, and the symbol of the function you want to probe. You can use tools like objdump, nm, or readelf to find the symbol of a function in a binary file. Alternatively, you can use the perf tool to list the available uprobes for a running process.
```

Comment:
This is already nice, here are some thoughts to improve this, you don't need to absolutely do that in that PR but we can do a followup:

those are nice explanations: could we put the tools name in backtick like `objdump`, `nm` and maybe put an example of how you can find the `readline` symbol in `/bin/bash` with `nm` or another one? Like with a little step showing:

```bash
nm /bin/bash | grep readline
```
and showing what the output would look like?